addons/project/migrations/9.0.1.1/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,187 @@
+---Fields in module 'project'---
+project      / project.project          / date (date)                   : NEW
+project      / project.project          / date_start (date)             : NEW
+project      / project.project          / label_tasks (char)            : NEW
+# New fields ---> Nothing to do
+
+project      / project.project          / members (many2many)           : DEL relation: res.users
+# Done ---> Field removed, no more project team, so renamed column to None to allow recovery with a future OCA module
+
+project      / project.project          / message_follower_ids (many2many): not a function anymore
+project      / project.project          / message_follower_ids (many2many): relation is now 'mail.followers' ('res.partner')
+project      / project.project          / message_follower_ids (many2many): type is now 'one2many' ('many2many')
+# Concerns module Mail (has to be done in another migration script specifically for that module) ---> Nothing to do
+
+project      / project.project          / state (selection)             : selection_keys is now '['cancelled', 'close', 'draft', 'open', 'pending']' ('['cancelled', 'close', 'draft', 'open', 'pending', 'template']')
+# Done ---> 'template' is converted to 'draft'
+# TODO - does this behavior make sense on the functional level?
```

Comment:
Looks like template functionality has been removed. Set existing templates to active=False and/or state draft indeed.